src/main/webapp/business.js

```diff
@@ -1,42 +1,70 @@
-// implementing map
+function getInfo() {
+  fetch('/business').then(response => response.json()).then((json) => {
+    var counter = 0;
+    for (var i = 1; i < json.length; i++) {
+      if (json[i][1] == 'in-person') {  // makes list of in-person businessess
+        var inPersonList = document.getElementById('inpersonbus');
+        var listItem = document.createElement('button');
+        var innerText = json[i][0] + '-' + json[i][2];
+        listItem.type = 'button';
+        listItem.className = 'list-group-item list-group-item-action';
+        listItem.innerHTML = innerText;
```

Comment:
'button' & 'list-group-...' can be const and can be defined outside the loop

Reply:
I understand why const is a good thing, but since I only use it once is it easier to leave it in the loop for readibility?